### PR TITLE
OCPBUGS-38249: Verify that the codegen changes were commited

### PR DESCRIPTION
**Pull request by kyrtapz**
In the past we had several bugs in which we forgot to run `hack/update-codegen.sh` that resulted in the APIs not being updated.
This change ensures that this won't happen as CI would catch that in the `verify` step. 
Tested here: https://github.com/openshift/cluster-network-operator/pull/2176

### CODE REVIEW

File: manifests/0000_70_cluster-network-operator_01_egr_crd.yaml [hunk 1]
```diff
@@ -7,7 +7,6 @@ metadata:
     api.openshift.io/merged-by-featuregates: "true"
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
-    include.release.openshift.io/single-node-developer: "true"
```
Comment (kyrtapz) [inline manifests/0000_70_cluster-network-operator_01_egr_crd.yaml:10]: Note: `single-node-developer` is no longer a valid cluster profile

**Comment by martinkennelly** [comment]
/lgtm
/hold

Holding to allow other PR review volunteers to review.

**Comment by jcaamano** [comment]
/retitle OCPBUGS-38249: Verify that the codegen changes were commited

**Comment by jcaamano** [comment]
/override ci/prow/okd-scos-images

we don't know what is going on with okd images, tracking here
https://issues.redhat.com/browse/OCPBUGS-38256

**Comment by jcaamano** [comment]
/override ci/prow/okd-scos-images

we don't know what is going on with okd images, tracking here
https://issues.redhat.com/browse/OCPBUGS-38256

**Comment by kyrtapz** [comment]
/test okd-scos-images
:crossed_fingers:

**Pull request merged**